packages/google_sign_in/google_sign_in_web/lib/src/generated/gapiauth2.dart

```diff
@@ -18,14 +10,24 @@ import "package:js/js_util.dart" show promiseToFuture;
 
 /// <reference types="gapi" />
 
+// https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/gapi.auth2
```

Comment:
After all the manual changes to these files, I think we can safely move them out of the "generated" directory...